qemu/qemu.go

```diff
@@ -825,6 +825,41 @@ func (vfioDev VFIODevice) QemuParams(config *Config) []string {
 	return qemuParams
 }
 
+// SCSIController represents a SCSI controller device.
+type SCSIController struct {
+	ID   string
+	Bus  string
+	Addr string
+}
+
+// Valid returns true if the SCSIController structure is valid and complete.
+func (scsiCon SCSIController) Valid() bool {
+	if scsiCon.ID == "" {
```

Comment:
```Bus``` and ```Addr``` are optional ? could you comment it in the ```struct```?

Reply:
@devimc comments added.